airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/asserts.py

```diff
@@ -18,6 +18,11 @@
                               r".*$"))  # timezone
 # fmt: on
 
+# In Json schema, numbers with a zero fractional part are considered integers. E.G. 1.0 is considered a valid integer
+# For stricter type validation we don't want to keep this behavior. We want to consider integers in the Pythonic way.
+strict_integer_type_checker = Draft7Validator.TYPE_CHECKER.redefine("integer", lambda _, value: isinstance(value, int))
```

Comment:
oh nice, it's cool that jsonschema lets you redefine the validators like this 👍🏼